languages/julia/concepts.csv

```diff
@@ -0,0 +1,74 @@
+concept,category
```

Comment:
I think this document would perhaps make more sense in the `reference` directory, which also contains the document with the Concepts and the Concept extraction documents.